cmd/node/config/config.toml

```diff
@@ -607,6 +607,7 @@
 
 [StateTriesConfig]
     CheckpointRoundsModulus = 100
+    CheckpointsEnabled = true
```

Comment:
is the default true ? with checkpoints ?

Reply:
Will discuss and will eventually change on the testnet/devnet/mainnet if this should be set to false.